src/ahrens_lab_to_nwb/yu_mu_cell_2019/yu_mu_cell_2019_segmentation_extractor.py

```diff
@@ -22,14 +22,28 @@ def __init__(self, file_path: FilePathType, sampling_frequency: float):
         self.file_path = file_path
         self._file = h5py.File(name=file_path)
 
-        baseline_shape = self._file["baseline"].shape
+        # Automatic detection of differing formats
+        if "baseline" in self._file:  # Dual-color sessions
+            self._baseline_group_name = "baseline"
+            self._timeseries_group_name = "timeseries"
+            self._pixel_mask_name_map = dict(x="x", y="y", z="z")
+
+            self._image_shape = (2048, 2048, 29)
+        else:  # Single-color sessions
+            self._baseline_group_name = "Cell_baseline"
+            self._timeseries_group_name = "Cell_timesers"
+            self._pixel_mask_name_map = dict(x="Cell_X", y="Cell_Y", z="Cell_Z")
```

Comment:
Single-color segmentation data has a slightly different naming convention. They're saved as doubles by default though, so no need to manually adjust them